fmriprep/interfaces/utils.py

```diff
@@ -34,23 +40,113 @@ def _run_interface(self, runtime):
         return runtime
 
 
-def prepare_roi_from_probtissue(in_file, epi_mask, epi_mask_erosion_mm=0,
-                                erosion_mm=0):
-    import os
-    import nibabel as nb
-    import scipy.ndimage as nd
-    from nilearn.image import resample_to_img
+class TPM2ROIInputSpec(BaseInterfaceInputSpec):
+    in_file = File(exists=True, mandatory=True, desc='input file')
+    in_mask = File(exists=True, mandatory=True, desc='input mask')
+    mask_erode_mm = traits.Float(0.0, usedefault=True,
+                                 desc='erode input mask (kernel width in mm)')
+    erode_mm = traits.Float(0.0, usedefault=True,
+                            desc='erode output mask (kernel width in mm)')
+    pthres = traits.Float(0.95, usedefault=True,
+                          desc='threshold for the tissue probability maps')
```

Comment:
I'd prefer this be something like `probability_threshold`, or `prob_thresh` or something a little more explicit. The main place we'll be reading it is

```Python
node = pe.Node(TPM2ROI(probability_threshold=0.95), ...)
```